.github/workflows/build.yaml

```diff
@@ -0,0 +1,56 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+
+  pull_request:
+  workflow_call:
+    outputs:
+      windows:
+        description: "Windows binaries"
+        value: ${{jobs.build_windows.outputs.artifact}}
+      linux:
+        description: "Linux binaries"
+        value: ${{jobs.build_windows.outputs.artifact}}
```

Comment:
I think this line was the mistake @thesayyn  :)

Reply:
i am not good at multitasking i guess :/